graylog2-web-interface/src/components/search/MessageFields.jsx

```diff
@@ -19,7 +19,7 @@ const MessageFields = React.createClass({
 
   _formatFields(fields, showDecoration) {
     if (!showDecoration || !this.props.message.decoration_stats) {
-      return Object.keys(fields).sort().map(key => <MessageField {...this.props} fieldName={key} value={fields[key]} />);
+      return Object.keys(fields).sort().map(key => <MessageField key={key} {...this.props} fieldName={key} value={String(fields[key])} />);
```

Comment:
Looking at the code, `MessageFieldDescription` only uses the field value to add it to the search bar if that action is required. In the end `possiblyHighlight()` is converting and returning the value we want to display already, so I don't think we need to convert anything at this point. I will simply relax the prop as you suggested in the first place.